src/com/tictactoe/tictactoegame/TicTacToeGame.java

```diff
@@ -0,0 +1,212 @@
+        cellChoice();
+        cellPlacement (player.symbol);
+        System.out.println(this);
+        return endOfGame(player);
+    }
+
+    /**
+     * asks the player the number of his chosen cell
+     * checks if number OK (if it's an Integer, (if it exists and if this cell isn't already filled) thanks to the validCoor()method)
+     * calculates line and column
+     */
+    public void cellChoice() {
+        var coorIsValid= false;
+
+        do {
+            System.out.println("Choisissez une case (entre 1 et 9): ");
+            var scanner = new Scanner(System.in);
+            try {
+                String entry=scanner.nextLine();
+                if (entry.equals("exit")){
+                    System.out.println("Vous quittez la partie.");
+                    System.exit(0);
+                } else {
+                    coor = Integer.parseInt(entry);
+
+                }
+                line =lineCalcFunc(coor);
+                col=colCalcFunc(coor);
+
+                coorIsValid=validCoor();
+            }
+            catch (NumberFormatException e){
+                System.out.println("Attention, ce n'est pas un entier...");
+            }
+            catch (TictactoeMismatchInputException t){
+                System.out.println(t.getMessage());
+            }
+
+        }while (!coorIsValid);
+
+    }
+
+    /**
+     * checks if cells exists and isn't already filled
+     * @return true if no problem
+     * @throws TictactoeMismatchInputException (if celle doesn't exist or is filled)
+     */
+    public boolean validCoor() throws TictactoeMismatchInputException{
+        if (coor<1||coor>9){
+            throw new TictactoeMismatchInputException("Ceci n'est pas le numéro d'une case, vous ne pouvez pas la choisir.");
+        }else {
+            if (gameMap[line][col]=='X'||gameMap[line][col]=='O'){
+                throw new TictactoeMismatchInputException("Cette case a déjà été choisie... Concentrez-vous un peu, que diable!");
+            }
+        }
+        return true;
+    }
+
+    /**
+     * places the right symbol in the chosen cell
+     * @param symbol : current player's cell
+     */
+    public void cellPlacement(char symbol){
+        gameMap[line][col]=symbol;
+    }
+
+    /**
+     * checks if game is over by calling two methods victoryChecking() and exAequo();
+     * @param player : Object with attributes symbol and name
+     * @return true if game is over
+     */
+    public boolean endOfGame(Player player ) {
+        if (victoryChecking(player.symbol)){
+            System.out.println("Nous avons un winner! : "+ player.name+ " a gagné!");
+        }
+        return (victoryChecking(player.symbol)) || exAequo();
+    }
+
+    /**
+     * Checks all the possible alignments
+     * @param symbol : current player's symbol
+     * @return true if there is at least one alignment OK (stops at the first found alignment)
+     */
+    public boolean victoryChecking(char symbol){
```

Comment:
An alternative implementation is to 
 * Create an array of 3 coords 
 * Compute and push all lines / columns / diagonals in the array (you may want to use static method)
 * Check if any 3 coords in the array is a win 

This way, you don't need to make specific code for line, columns, and diagonals.